github-dark.css

```diff
@@ -4561,7 +4561,7 @@
   .site-header-nav .dropdown-menu a.active,
   .site-header-nav .dropdown-menu a .octicon-check,
   .marketplace-product-callout .branch-action-item-icon,
-  .user-status-org-button span.text-bold {
+  .user-status-org-button span.text-bold, .markdown-body .octicon {
```

Comment:
This is not needed =)

Reply:
Without this, won't octicon colours change from ``#dcdcdc`` to ``#bebebe``?

In help pages, octicons seem to always be used alongside bolded text (which use ``#dcdcdc``) or used on their own in place of where bolded text would've usually been. A similar colour difference also exists in GitHub's default theme (where octicons are ``#000000`` and normal/bolded text is ``#24292e``).

Removing this line's change would deviate from that colour pattern.

Thoughts?

Reply:
Add like below then, `markdown-body span.octicon svg`